lib/fhir/resources/related-person.js

```diff
@@ -29,49 +30,8 @@ module.exports = (mongo) => {
           return callback(null, fhirCommon.buildHTTPOutcome(400, 'error', 'invalid', badRequest.message))
         }
 
-        let query = { $and: [] }
-
-        if (queryObject['_id']) {
```

Comment:
Are `_id` query params handled for all resources generically? I think we should verify this.

Reply:
Yip that should be a default supported param. This one can be removed